### ISSUE TITLE
SOGo change autoBox.html content

### ISSUE THREAD

**Issue by prononext** [mailcow/mailcow-dockerized#5306]
### Summary

It would be really nice to have a possiblity to change the autoBox.html content of the SOGo login. 

![image](https://github.com/mailcow/mailcow-dockerized/assets/24569958/01d46ea3-5b1c-4488-addb-1b24bad2e351)

![image](https://github.com/mailcow/mailcow-dockerized/assets/24569958/3b1e13fc-ed80-463b-944c-8224488c4548)

### Motivation

Like this legally required links could be added like privacy policy, usage therms etc.

**Comment by MAGICCC** [comment]
As SOGo is not developed by mailcow you have limited possibilities.
You can check the FAQs - https://www.sogo.nu/support/faq/how-to-customize-the-html.html
Or ask in their bugtracker for an easier solution: https://bugs.sogo.nu/my_view_page.php

**Comment by prononext** [comment]
I thought because mailcow allows customizing other parts oft the sogo there could be a way to customize the .wox also, outside oft the Sogo Volumen/Container?

https://docs.mailcow.email/manual-guides/SOGo/u_e-sogo/

**Comment by MAGICCC** [comment]
Yes that's possible too, but would require some work.
You can either edit https://github.com/Alinto/sogo/blob/master/UI/Templates/MainUI/SOGoRootPage.wox
Or directly the [AboutBox](https://github.com/Alinto/sogo/blob/master/UI/MainUI/English.lproj/Localizable.strings#L80)
But you may have to do it in all languages if you want to do it
https://github.com/Alinto/sogo/tree/master/UI/MainUI

Restart sogo container and it's working.
Indeed it's a hassle

**Comment by prononext** [comment]
Sounds nice, but where do I place my custom .wox file? And is anything else needed?

**Comment by MAGICCC** [comment]
You can use the docs for an example, just with a different .wox. But you have to edit the language files and not the .wox itsself
Also do note if Alinto changes something inside the language files you have to merge your changes with theirs

**Comment by prononext** [comment]
@MAGICCC  Sorry for the question, but it is still not clear where I place my language file? Can I place it in the /opt/mailcow-dockerized/data/conf/sogo? And could I also place a .wox there if I wanted a custom one?

**Comment by MAGICCC** [comment]
Yes you can put it anywhere as long as the path inside the container is right

**Comment by prononext** [comment]
@MAGICCC  I am very sorry but it is not clear at all:

> Yes you can put it anywhere as long as the path inside the container is right

Where exactly inside the mailcow-dockerized directory do I store the custom sogo .wox file or  language file? 
So that after restarting my customizations are reflected in my SOGo.

**Comment by MAGICCC** [comment]
It doesn't matter, you need to find the right path inside the container only

**Comment by prononext** [comment]
ah ok. so there is no way to get it placed into /opt/mailcow-dockerized/conf/sogo ... outside of the container
Hopefully that will be supported soon, for the SOGo .wox and languages files.

**Comment by MAGICCC** [comment]
Maybe you misunderstand, you have to place it outside of the container and link the file inside, as explained in the docs....
It doesn't matter where you place it outside.

**Comment by patschi** [comment]
There were instructions posted and were SOGo-specific. Due to issue age and provided instructions, I'll close this one.
